_pages/oidc.md

```diff
@@ -133,12 +133,15 @@ https://idp.int.identitysandbox.gov/openid_connect/authorize?
   The unique identifier for the client. This will be registered with the Login.gov IdP in advance.
 
 * **code_challenge** — *required for PKCE*
-  The URL-safe base64 encoding of the SHA256 digest of a random value generated by the client. The original random value is referred to as the `code_verifier` and is later used with the token endpoint. Generating these values in Ruby might look like this, for example:
+  The URL-safe Base64 encoding of the SHA256 digest of a random value generated by the client. The original random value is referred to as the `code_verifier` and is later used with the token endpoint. Generating these values in Ruby might look like this, for example:
```

Comment:
Should we hyperlink to the RFC from this section?

Reply:
yep, can do